### DISCUSSION TITLE
Plans on adding Matrix Core Cosmetics?

### DISCUSSION THREAD

**Discussion by Visconde-dev** [AnthonyMichaelTDM/DRG-Save-Editor#5] [Q&A] [answered]
Hello! I found no way to contact you , but i simply wanted to ask that simple question. Since you've made weapons overclocks work , i wonder if it would be possible to make Cosmetic matrix cores available on the forge? And, if those things are on the table? Thanks for the patience in advance.

**Comment by AnthonyMichaelTDM** [comment, marked as answer]
Cosmetic overclocks are possible, they're stored in a very similar way as weapon overclocks, the one issue is the sheer quantity of cosmetics in the game, even with save files it'd be hours of repetitive work to find the GUID's of every in-game cosmetic.

Because of this I currently don't plan on adding adding cosmetic OC's, however it's not necessarily off the table if more people are willing and able to help.


That is to say, if someone were to provide the GUIDs for all the cosmetic OC's, I'd be more than willing to incorporate them into an editor.
